crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampPresenterCoordinator.kt

```diff
@@ -32,6 +42,7 @@ internal class OnrampPresenterCoordinator @Inject constructor(
     linkController: LinkController,
     lifecycleOwner: LifecycleOwner,
     private val activity: ComponentActivity,
+    private val cryptoApiRepository: CryptoApiRepository,
```

Comment:
I think adding this dependency as well as pulling the CSCS from the interactor are code smells. We should aim to keep this Coordinator class dumb (like an Activity/View -- mostly wiring) and move the logic to `OnrampInteractor`.